travis-ci/run-standard-tests.sh

```diff
@@ -41,33 +35,55 @@ else
     ulimit -Sn 8192
 fi
 
-# Test
-if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+if [[ $GXBUILD ]]; then
```

Comment:
There needs to be a similar if statement for `GO111MODULE` that runs `go mod download` so that only modules listed in the `go.sum` file are downloaded for the build. Also, currently downloading of go module dependencies isn't handled at all in this script.

Reply:
With `GO111MODULE=on` set, all `go` commands download `go.mod` dependencies and verify them. I don't see any need for explicit download step.
What do you think?